fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/ManagedRdf.java

```diff
@@ -35,12 +41,25 @@
 
     private static final Model model = createDefaultModel();
 
+    private static final Set<URI> managedTypes = of(
```

Comment:
Would it be possible to try to reconcile the following as opposed to propagating more flavors of managed and internal and protected, etc:

FedoraTypesUtils.isProtectedAndShouldBeHidden
FedoraTypesUtils.EXPOSED_PROTECTED_JCR_TYPES
FedoraTypesUtils.isSuppressedProperty
FedoraTypesUtils.isInternalProperty
FedoraTypesUtils.isInternalNode

ManagedRdf.isManagedTriple
ManagedRdf.isManagedMixin

RdfLexicon.isManagedPredicate
RdfLexicon.isManagedPredicateURI
RdfLexicon.managedProperties
RdfLexicon.isManagedNamespace
RdfLexicon.managedNamespaces